src/cursor/cur_backup.c

```diff
@@ -159,8 +159,6 @@ __wt_backup_file_remove(WT_SESSION_IMPL *session)
      * there's any chance of an incremental backup file existing.
      */
```

Comment:
New line 156-160 doesn't make any sense without a backup or source file being removed.

Reply:
Good catch

Reply:
Thanks! Fixed in 3f74a24.